hermes-api/src/main/java/pl/allegro/tech/hermes/api/Anonymizable.java

```diff
@@ -0,0 +1,5 @@
+package pl.allegro.tech.hermes.api;
+
+public interface Anonymizable {
+    Anonymizable anonymize();
```

Comment:
i don't like this interface, feels like it enters the domain to fix this single problem in this single context and adds no additional value - it returns an `Anonymizable` instance, which probably won't be useful. When i anonymize `Subscription` i want to get back a subscription, not a very generic interface.